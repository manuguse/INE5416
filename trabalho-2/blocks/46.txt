1 2 7 10 13 13 14 19 19 23
1 2 7 7 14 14 14 20 19 19
2 2 2 7 14 14 14 19 19 19
3 3 2 2 15 15 15 21 21 21
3 3 8 11 15 15 15 21 18 24
4 4 6 11 12 15 18 18 18 24
5 6 6 12 12 16 16 18 18 24
5 6 6 12 12 12 16 17 18 24
5 5 6 12 9 17 17 17 17 22
5 5 9 9 9 17 17 22 22 22