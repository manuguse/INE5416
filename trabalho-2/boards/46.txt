- 3 4 - - 1 - - 6 -
- 2 - - - - - - - 3
- - - - 1 - 5 - 4 -
3 - - 7 6 7 - - - 3
- - - - 3 - - - 7 -
- - - - 6 - - 5 - -
6 - 4 - - - 3 - 3 -
- - - - - 7 - - 2 -
- - - - - - 6 2 - -
3 - - 2 1 4 - 4 - 2