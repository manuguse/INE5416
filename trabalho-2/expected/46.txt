2 3 4 1 2 1 7 2 6 1
1 2 1 3 4 3 6 1 5 3
4 1 6 2 1 2 5 7 4 1
3 4 5 7 6 7 2 4 1 3
1 2 1 2 3 5 1 2 7 4
2 1 6 1 6 4 6 5 4 3
6 5 4 5 2 1 3 1 3 2
5 3 2 4 1 7 2 3 2 1
4 2 1 3 4 5 6 2 7 3
3 1 3 2 1 4 1 4 1 2